app/controllers/stripe-setup/bank-details/get.controller.js

```diff
@@ -1,7 +1,19 @@
 'use strict'
 
 const { response } = require('../../../utils/response')
+const paths = require('../../../paths')
+const formatAccountPathsFor = require('../../../utils/format-account-paths-for')
 
-module.exports = (req, res) => {
+module.exports = (req, res, next) => {
+  const stripeAccountSetup = req.account.connectorGatewayAccountStripeProgress
+
+  if (!stripeAccountSetup) {
+    return next(new Error('Stripe setup progress is not available on request'))
+  } else {
```

Comment:
if you wanted to save an extra few lines and some indentation, you could remove the `else` as we return if the `stripeAccountSetup` isn't present.

Reply:
Good spot! will update in next PR.